ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -175,10 +175,20 @@ void RSkComponentTextInput::OnPaint(SkCanvas *canvas) {
 * @return      True if key is handled else false
 */
 
-void RSkComponentTextInput::onHandleKey(rnsKey eventKeyType, bool keyRepeat, bool *stopPropagation) {
+void RSkComponentTextInput::onHandleKey(rnsKey eventKeyType, bool keyRepeat, rnsKeyAction keyAction,bool *stopPropagation) {
   RNS_LOG_DEBUG("[onHandleKey] ENTRY");
   *stopPropagation = false;
   if (!editable_) {
+    if(keyAction == RNS_KEY_Release && eventKeyType == RNS_KEY_Select){
```

Comment:
better to add eventKeyType == RNS_KEY_Select check first

Reply:
I feel, it is good to put conditions in parentheses for improved readability.